modules/java_api/org/intel/openvino/Core.java

```diff
@@ -47,10 +47,9 @@ private static String getLibraryName(String name, String linux_ver) {
         return name;
     }
 
-    /**
-     * Use this method to initialize native libraries. Other files like plugins.xml and *.mvcmd will
-     * be also copied to temporal location which makes them visible in runtime.
-     */
+    // Use this method to initialize native libraries and other files like
+    // plugins.xml and *.mvcmd in case of the JAR package os OpenVINO.
+    // To Do: fix for Android!
```

Comment:
Why docstring has been changed from `/**/` to `//`?

Reply:
Thanks, fixed